common/api/map-layers-formats.api.md

```diff
@@ -82,6 +82,10 @@ export class MapFeatureInfoTool extends PrimitiveTool {
     onRestartTool(): Promise<void>;
     // (undocumented)
     requireWriteableTarget(): boolean;
+    // @internal (undocumented)
+    protected setupAndPromptForNextAction(): void;
+    // (undocumented)
+    protected showPrompt(): void;
```

Comment:
Should this be internal too since it's only called in an internal method?

Reply:
I agree, fixed.